book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :tittle, :author
+  attr_reader :rentals
+
+  def initialize(tittle, author)
+    @tittle = tittle
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Kindly create a method that creates a new rental object of class Rental and passes `date`, `person`, and `book` itself as a parameter to achieve the `has-many` association. You may learn more about both ways association from [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).

Reply:
thanks for the message @gemmen29 It's done!